dashboard/config/routes.rb

```diff
@@ -393,6 +393,10 @@ module OPS
     post 'attend/:session_code/join', controller: 'workshop_enrollment', action: 'confirm_join_session'
     get 'attend/:session_code/upgrade', controller: 'session_attendance', action: 'upgrade_account'
     post 'attend/:session_code/upgrade', controller: 'session_attendance', action: 'confirm_upgrade_account'
+
+    get 'workshop_user_admin/facilitator_courses', controller: 'workshop_user_admin', action: 'facilitator_courses_form'
+    post 'workshop_user_admin/assign_course', controller: 'workshop_user_admin', action: 'assign_course'
+    get 'workshop_user_admin/remove_course', controller: 'workshop_user_admin', action: 'remove_course'
```

Comment:
I believe linking to a `delete` route in the view would require some client-side JavaScript?